accounts-cluster-bench/src/main.rs

```diff
@@ -12,23 +12,24 @@ use {
         rpc_client::SerializableTransaction, rpc_config::RpcBlockConfig,
         rpc_request::MAX_GET_CONFIRMED_BLOCKS_RANGE, transaction_executor::TransactionExecutor,
     },
+    solana_clock::Slot,
+    solana_commitment_config::CommitmentConfig,
     solana_gossip::gossip_service::discover,
+    solana_hash::Hash,
+    solana_instruction::{AccountMeta, Instruction},
+    solana_keypair::{read_keypair_file, Keypair},
     solana_measure::measure::Measure,
+    solana_message::Message,
+    solana_program_pack::Pack,
+    solana_pubkey::Pubkey,
     solana_rpc_client::rpc_client::RpcClient,
     solana_rpc_client_api::request::TokenAccountsFilter,
-    solana_sdk::{
-        clock::Slot,
-        commitment_config::CommitmentConfig,
-        hash::Hash,
-        instruction::{AccountMeta, Instruction},
-        message::Message,
-        program_pack::Pack,
-        pubkey::Pubkey,
-        signature::{read_keypair_file, Keypair, Signature, Signer},
-        system_instruction, system_program,
-        transaction::Transaction,
-    },
+    solana_sdk_ids::system_program,
+    solana_signature::Signature,
+    solana_signer::Signer,
     solana_streamer::socket::SocketAddrSpace,
+    solana_system_interface::instruction as system_instruction,
```

Comment:
```suggestion
    solana_system_interface::{instruction as system_instruction, program as system_program},
```

then you can drop the `solana-sdk-ids` dep